Automattic/wp-calypso
# Consistent state handling of domain whois data.

#### Changes proposed in this Pull Request

When the contact information for a domain is changed on the Edit Contact Info page, the correct email that the verification has been sent to isn't reflected on the Domain Settings page because the two pages are using different data to display the email address.

#### Testing instructions

- Pick a domain that we manage the emails for.
- Update the email address in the Edit Contact Info page.
- Return to the Domain Settings page.
- Make sure that the ICANN notice at the top of the page shows the newly saved email.
- Make sure that the email is sent to the correct address.
- Make sure that the contact form still works correctly for newly purchased domains as well.
